lib/api/helpers.js

```diff
@@ -200,6 +201,26 @@ export function getSlug(slugString) {
   return slug;
 }
 
+/**
+ * formatPhoneNo
+ * @summary prepends country code to phone no. if required
+ * @param  {String} phone the original phone no.
+ * @param  {String} countryCode the country's code to which the phone no. belongs to
+ * @return {String} the phone no. with country extension.
+ */
+export function formatPhoneNo(phone, countryCode) {
+  try {
+    // phone no. already has the country code attached
+    if (isValidNumber(phone)) {
+      return phone;
+    }
+    // try attaching the country code to phone no.
+    return `${getPhoneCode(countryCode)}${parse(phone, countryCode).phone}`;
+  } catch (error) {
```

Comment:
@zenweasel asked how will the caller know that a error has occurred.
Answer -> This is a best effort function, so it tries to add the country code to the phone number, but if that is not possible it just returns the original number.

Reply:
Not sure what is meant "best effort function" but I would say that we want to go with how people would [expect](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) a function to behave. And I would say that if I submit something that is not a phone number to a `formatPhoneNumber` function and it can't format it, I would expect an error, not to just get back my bad phone number untouched. To just do nothing and swallow the error seems like it's just asking for errors upstream when you try to submit it to SMS sending.